spec/views/insured/families/_shop_for_plans_widget.html.erb_spec.rb

```diff
@@ -55,8 +55,8 @@
       render "insured/families/shop_for_plans_widget"
     end
 
-    it "should have link without change_plan" do
-      expect(rendered).to have_selector("a[href='/insured/consumer_role/build']")
+    it "should not have link without change_plan" do
```

Comment:
I'd rather have something using contexts.
`
context "change plan link can be enabled or disabled" do
  it "shows a link when individual market is enabled" do
     allow(view).to receive(:individual_market_is_enabled?).and_return(true)
     expect(rendered) ....
   end
   it "does not show a change plan link if individual market is disabled" do
     ...
   end
end
`

something like that